assets-and-images.md

```diff
@@ -206,3 +206,52 @@ level classes, like
 or
 [`ImageCache`](https://docs.flutter.io/flutter/services/ImageCache-class.html),
 you'll also notice parameters related to scale.)
+
+## Platform assets
+
+There will also be occasions to work with assets in the platform projects directly. Below are two common cases where assets are used before the Flutter framework is loaded and running.
+
+### Updating the app icon
+Updating your Flutter application's launch icon works the same way as updating launch icons in native Android or iOS applications.
+
+![Launch icon](/images/assets-and-images/icon.png)
+
+#### Android
+
+In your Flutter project's root directory, navigate to `.../android/app/src/main/res`. The various bitmap resource folders such as `mipmap-hdpi` already contain placeholder images named `ic_launcher.png`. Simply replace them with your desired assets respecting the recommended icon size per screen density as indicated by the [Android Developer Guide](https://developer.android.com/guide/practices/ui_guidelines/icon_design_launcher.html#size).
+
+![Android icon location](/images/assets-and-images/android-icon-path.png)
+
+<aside class="alert alert-info" markdown="1">
+**Note:** if you rename the .png files, you must also match the new name in your `AndroidManifest.xml`'s `<application>` tag's `android:icon` attribute.
+</aside>
+
+#### iOS
+
+In your Flutter project's root directory, navigate to `.../ios/Runner/Assets.xcassets/AppIcon.appiconset`. The directory already contains placeholder images. Simply replace them with the appropriately sized images as indicated by their filename as dictated by the Apple [Human Interface Guidelines](https://developer.apple.com/ios/human-interface-guidelines/graphics/app-icon). Keep the original file names.
```

Comment:
Due to https://github.com/flutter/flutter/issues/10153, long backtick strings like this will cause the viewport to no longer fit on mobile devices:

![screen shot 2017-08-14 at 14 29 14](https://user-images.githubusercontent.com/13644170/29271686-fec99946-80fc-11e7-9082-3d1db06ce2cc.png)


I've found the easiest workaround for now to be to rewrite sentences like:

    navigate to `.../ios/Runner/Assets.xcassets/AppIcon.appiconset`.

 to:

    navigate to `AppIcon.appiconset` located in .../ios/Runner/Assets.xcassets/.